src/notifications/contracts/mongo-database-example.js

```diff
@@ -4,12 +4,16 @@ import NotificationDatabaseContract from './database';
 import Notification from './mongo-model-example';
 
 class MongoNotificationDatabase extends NotificationDatabaseContract {
+  constructor() {
```

Comment:
Yup you're right. Same comment as before, we were passing the connection parameters on the constructor of this class to open the connection to mongo right away. But that is not being used that way right know